authorizer_lambda/src/authorizer_lambda/main.py

```diff
@@ -1,19 +1,61 @@
 from typing import Any
+
+from authorizer_lambda.cognito_adapter import fetch_owner_from_cognito
 from authorizer_lambda.config import Settings, config
 from authorizer_lambda.models import (
     AuthorizationResponse,
     OwnerResponse,
     TokenAuthorizationRequest,
     OwnerRequest,
+    PolicyDocument,
+    PolicyStatement,
+    StatementEffect,
 )
+import jwt
 
 
 def get_owner_uuid(cognito_id: str, settings: Settings) -> str:
     """Retrieve the owner_uuid from the lambda based on the cognito id and return it."""
     request = OwnerRequest(cognito_id=cognito_id)
+    cognito_object = fetch_owner_from_cognito(cognito_id)
```

Comment:
I think you misunderstood how to get the owner_uuid. Apologies if the instructions weren't clear enough. But as the comment on line 21 says, you were supposed to get the owner_uuid by calling a lambda, who's arn is configured in `config.backend_lambda_arn` already. So something like

```
client = boto3.client("lambda")
result = client.invoke(
    FunctionName=
    InvocationType="RequestResponse",
    Payload=OwnerRequest(cognito_id=cognito_id).json().encode("utf-8"),
)
payload = OwnerResponse.parse_obj(json.load(result["Payload"]))
return response.owner.owner_uuid
```
And in the unittests you can mock `boto3`

Reply:
I remember that part, but I misunderstood when you said I could mock that call. I thought that doing a separate module with the name of `adapter` would be enough. Totally my mistake, sorry.